packages/unified-pipeline/src/components/Canvas/elements/Nodes/GroupNode/GroupNode.tsx

```diff
@@ -213,10 +213,10 @@ export default function GroupNode(props: NodeProps<GroupNodeProps>) {
           height
         }}
         className={cx(
-          'flex flex-col items-center justify-between text-xs font-medium leading-3 box-border text-left p-2.5 rounded-lg bg-[rgba(15,16,17,1)] border border-[rgba(48,51,54,1)] border-dashed',
+          'flex flex-col items-center justify-between text-xs font-medium leading-3 box-border text-left p-2.5 rounded-lg bg-studio-6 border border-studio-5 border-dashed',
```

Comment:
we really shouldn't have 10 unique shades of gray just for the studio.
cant we use existing colors anywhere here?

Reply:
Pixelpoint has been using different greys in the design for Pipeline Studio